src/Domain/Item/Item.ts

```diff
@@ -6,6 +6,7 @@ import { Revision } from '../Revision/Revision'
 @Index('user_uuid_and_updated_at_timestamp_and_created_at_timestamp', ['userUuid', 'updatedAt', 'createdAt'])
 export class Item {
   static readonly CONTENT_TYPE_NOTE = 'Note'
+  static readonly CONTENT_TYPE_ITEMS_KEYS = 'SN|ItemsKey'
```

Comment:
I think either singularize this to be `CONTENT_TYPE_ITEMS_KEY`, or better yet create an enum so we can have `ContentType.ItemsKey` :)

Reply:
Will fix 👍